docs/user/03-60-formatting-service-binding-secret.md

```diff
@@ -0,0 +1,232 @@
+To show both credentials returned from the service broker and additional **ServiceInstance** attributes as a JSON object, use the **secretRootKey** attribute in the service binding spec.
+
+The **secretRootKey** value is the name of the key that stores both credentials and service instance info. The credentials are represented in both formats: YAML or JSON.
+See the following examples:
+
+* Service binding
+
+  ```yaml
+  apiVersion: services.cloud.sap.com/v1
+  kind: ServiceBinding
+  metadata:
+    name: my-binding
+  spec:
+    serviceInstanceName: my-service-instance
+    secretRootKey: myCredentialsAndInstance
+  ```
+* Secret
+
+  ```yaml
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: my-binding
+  stringData:
+      myCredentialsAndInstance:
+          uri: https://my-service.authentication.eu10.hana.ondemand.com,
+          client_id: admin,
+          client_secret: ********,
+          instance_guid: my-service-instance-guid, // The service instance id
+          instance_name: my-service-instance, // Taken from the service instance external_name field if set. Otherwise from metadata.name
+          plan: sample-plan, // The service plan name
+          type: sample-service, // The service offering name
+  ```
+## Custom Formats 
+
+For additional flexibility, you can model the Secret resources according to your needs. 
+To generate a custom-formatted Secret, use the **secretTemplate** attribute in the ServiceBinding spec.
+This attribute expects a Go template as its value. For more information, see [Go Templates](https://pkg.go.dev/text/template).<br>
+Ensure the template is in the YAML format and has the structure of a Kubernetes Secret. 
+
+In the provided Secret, you can customize the `metadata` and `stringData` sections with the following options:
+
+- `metadata`: labels and annotations
+- `stringData`: customize or utilize one of the available formatting options following the instructions in the [Overview](#overview) section.
+
+
+> [!NOTE]  
+> If you customize `stringData`, it takes precedence over the provided pre-defined formats.
+
+The provided templates are executed on a map with the following available attributes:
+
+| Reference         | Description                                |                                                                          
+|-----------------|--------------------------------------------|
+| **instance.instance_guid** |  The service instance ID.     |
+| **instance.instance_name** |  The service instance name.   |                                                
+| **instance.plan**   |  The name of the service plan used to create this service instance. |  
+| **instance.type**   |  The name of the associated service offering. |  
+| **credentials.attributes(var)**   |  The content of the credentials depends on a service. For more details, refer to the documentation of the service you're using. |  
+
+See two examples demonstrating the ServiceBinding and generated Secret resources:
```

Comment:
```suggestion
The following examples demonstrate the ServiceBinding and generated Secret resources:
```
(avoid mentioning the number; it might change and then you'd have to remember overwriting the number :D)